### PR TITLE
Add to onboarding reproduction logs

### PR DESCRIPTION
**Pull request by KenWuqianghao**
System setup:

OS: macOS Sonoma 14.4.1
Memory: 16GB
Chip: Apple M1 Pro
Python Version: 3.12.3
Java Version: 21.0.3
Maven: 3.9.6

Suggestion: I think linking try-it instead of getting-started is more suitable as I don't see a getting-started section on the read-me.

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -322,8 +316,8 @@ It turns out that optimizing for MRR@10 and MAP yields the same settings.
 
 Here's the comparison between the Anserini default and optimized parameters:
 
-| Setting                                         | MRR@10 |    MAP | Recall@1000 |
-|:------------------------------------------------|-------:|-------:|------------:|
+| Setting                                             | MRR@10 |    MAP | Recall@1000 |
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:319]: I think you introduced inconsistencies here? Old table seems fine to me?

File: docs/start-here.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ What's the problem we're trying to solve?
 
 This is the definition I typically give:
 
-> Given an information need expressed as a query _q_, the text retrieval task is to return a ranked list of _k_ texts {_d<sub>1</sub>_, _d<sub>2</sub>_ ... _d<sub>k</sub>_} from an arbitrarily large but finite collection
-of texts _C_ = {_d<sub>i</sub>_} that maximizes a metric of interest, for example, nDCG, AP, etc.
+> Given an information need expressed as a query _q_, the text retrieval task is to return a ranked list of _k_ texts {_d`<sub>`1`</sub>`_, _d`<sub>`2`</sub>`_ ... _d`<sub>`k`</sub>`_} from an arbitrarily large but finite collection
```
Comment (lintool) [inline docs/start-here.md:23]: nah, I think I like original better...

**Comment by lintool** [comment]
thanks for the de-linting. left comments.

File: docs/experiments-msmarco-passage.md (hunk 1 above)
Comment (KenWuqianghao) [reply docs/experiments-msmarco-passage.md:319]: Oops, I clicked on the file and my editor did some formatting automatically. I will fix that.

File: docs/start-here.md (hunk 2 above)
Comment (KenWuqianghao) [reply docs/start-here.md:23]: Yeah same here, I didn't mean to change that. My editor did that automatically for some reason. Will fix.

File: docs/experiments-msmarco-passage.md [hunk 3]
```diff
@@ -89,7 +86,7 @@ On the other hand, retrieval needs to be fast, i.e., low latency, high throughpu
 
 With the data prep above, we can now index the MS MARCO passage collection in `collections/msmarco-passage/collection_jsonl`.
 
-If you haven't built Anserini already, build it now using the instructions in [anserini#-getting-started](https://github.com/castorini/anserini#-getting-started).
+If you haven't built Anserini already, build it now using the instructions in [anserini#-try-it](https://github.com/castorini/anserini?tab=readme-ov-file#-try-it).
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:89]: I think "Installation" is the better link?

**Comment by KenWuqianghao** [comment]
@lintool I have made the changes accordingly. Please let me know if anything I didn't expect breaks lol

**Review by lintool — APPROVED** [review]

**Review by lintool — APPROVED** [review]

**Pull request merged**